miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditLibraryController.java

```diff
@@ -845,6 +866,10 @@ public ModelAndView editPropagateSamples(@PathVariable String sampleIds, ModelMa
           hasPlain = true;
         }
         LibraryDto library = new LibraryDto();
+        Library tempLibrary = new LibraryImpl();
+        tempLibrary.setSample(sample);
+        final String libraryAlias = namingScheme.generateLibraryAlias(tempLibrary);
```

Comment:
Alias generation should be handled in the Service layer unless it's necessary to display the alias before the library is saved?